src/BloomExe/Book/BookStorage.cs

```diff
@@ -866,6 +866,8 @@ private string SanitizeNameForFileSystem(string name)
 				name = name.Replace(c, ' ');
 			}
 			name = name.Trim();
+			if (name.StartsWith("."))
```

Comment:
Seems to me we might want a Trim() in there somewhere...leading and trailing white space in a filename is confusing. And then we need to do something to make sure the end result is not an empty (or all-white) string, if that isn't happening somewhere else. And of course that it's not identical to some existing book...that may be done elsewhere, too.